bin/lerna.js

```diff
@@ -1,11 +1,6 @@
 #!/usr/bin/env node
 
 var args = process.argv.slice(2);
-if (args.length > 1) {
```

Comment:
This looks good although I'd prefer if the `args` assertion was present when the command isn't `run`.